dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/MetricsMonitorTest.java

```diff
@@ -117,7 +117,7 @@ public void testMpMetrics10Monitor10Feature() throws Exception {
     	Log.info(c, testName, "------- Enable mpMetrics-1.0 and monitor-1.0: vendor metrics should not be available ------");
     	server.setServerConfigurationFile("server_mpMetric10Monitor10.xml");
     	server.startServer();
-    	Log.info(c, testName, server.waitForStringInLog("defaultHttpEndpoint-ssl",60000));
+    	Log.info(c, testName, server.waitForStringInLog("SRVE9103I",60000));
```

Comment:
If SRVE9103I is not found, the test will continue.  Should all of these `server.waitForStringInLog()` be wrapped with `Assert.assertNotNull()`?

Reply:
I've put another commit on the branch with the assert.